Doppler.BigQueryMicroservice/Repository/Classes/UserAccessByUserRepository.cs

```diff
@@ -73,6 +76,41 @@ public async Task<IReadOnlyList<UserAccessByUser>> GetAllByUserIdAsync(string ac
                 }
             }
         }
+
+        public async Task<bool> MergeEmailsAsync(string accountName, List<string> emails)
+        {
+            using (var connection = await base.CreateConnectionAsync())
+            {
+                try
+                {
+                    var dt = new DataTable("[dbo].[TypeEmail]");
+                    dt.Columns.Add("Email", typeof(string));
+
+                    foreach (var email in emails)
+                    {
+                        dt.Rows.Add(email);
+                    }
+
+                    var result = await connection.QueryAsync("EXEC [datastudio].SaveEmailsByAccounName @P1_Emails, @P2_AccounName", new
+                    {
+                        P1_Emails = dt.AsTableValuedParameter("[dbo].[TypeEmail]"),
+                        P2_AccounName = accountName
+                    });
+
+                    return true;
+                }
+                catch (Exception ex)
+                {
+                    base.BigQueryLogger.LogError(ex.Message);
```

Comment:
What do you think?
```suggestion
                    base.BigQueryLogger.LogError(ex, $"Emails can not be merged for {nameof(accountName)} {accountName}");
```

Reply:
I think it is easier to detect where the error came from, good catch